pkg/sentry/platform/ring0/kernel_arm64.go

```diff
@@ -16,6 +16,12 @@
 
 package ring0
 
+//go:nosplit
+func HaltAndGo()
```

Comment:
All public functions have to have comments. Where do we use them?

Reply:
Hi @avagin 
Done.